content/en/agent/faq/certificate_verify_failed-error.md

```diff
@@ -68,27 +70,36 @@ rm "C:\Program Files\Datadog\Datadog Agent\files\datadog-cert.pem"
 restart-service -Force datadogagent
 ```
 
-*Windows GUI*
+If your Agent is configured to use a proxy, see the additional step in the [dedicated section below](#additional-step-if-the-windows-agent-5-x-is-configured-to-use-a-proxy-or-the-curl-http-client).
+
+*Using the Windows GUI*
+
+Delete `datadog-cert.pem`. You can locate this file in:
+
+* Agent `>=5.12.0`:
+  * 64-bit Windows: `C:\Program Files\Datadog\Datadog Agent\agent\`
+  * 32-bit Windows: The Datadog Agent was not available for 32-bit Windows systems starting in Agent 5.12
+* Agent `<= 5.11.x`:
+  * 64-bit Windows: `C:\Program Files (x86)\Datadog\Datadog Agent\files\`
+  * 32-bit Windows: `C:\Program Files\Datadog\Datadog Agent\files\`
+
+Once the file is removed, restart the Datadog Service from the Windows Service Manager.
 
-Delete `datadog-cert.pem`. You can locate this in: `C:\Program Files\Datadog\Datadog Agent\agent\`. 
-(For 32-bit Agent versions `<= 5.11` on 64-bit windows, the location will be 
-`C:\Program Files(x86)\Datadog\Datadog Agent\files\`. For all other Agent versions `<= 5.11`
-the location is `C:\Program Files\Datadog\Datadog Agent\files\`).  Once removed, restart
-the Datadog Service from the Windows Service Manager.
+If your Agent is configured to use a proxy, see the additional step in the [dedicated section below](#additional-step-if-the-windows-agent-5-x-is-configured-to-use-a-proxy-or-the-curl-http-client).
```

Comment:
It's not really an additional step, right? It's a different step? For windows agent 5.12+, if they're replacing datadog-cert.pem, they don't need to delete it first (but it does not matter if they do).

For Windows Agent v5, `<= 5.11.x`, do they need to delete the datadog-cert.pem? or is adding the ca_certs option enough

Reply:
For Windows Agent `<= 5.11.x`, adding the `ca_certs` option is enough. We're only using `datadog-cert.pem` if this option is not set, as seen in this code block:
https://github.com/DataDog/dd-agent/blob/4d2b6eb26fc2583f24fd992463452ef555702231/config.py#L667-L670

(This code block hasn't changed since it was introduced in `3.6.0`)

Reply:
yes, that's fair, I'll change this